examples/mobile/UIComponents/components/etc/version/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="version-page">
+		<div class="ui-header" data-position="fixed">
+			<h1>
+				Version
```

Comment:
Mabe "About" instead of "Version"
In future we can add other info to this page like Android version, Chrome engine version etc..., which will allow us to more easily debug issues